projects/common-form-elements/src/lib/dynamic-dropdown/dynamic-dropdown.component.css

```diff
@@ -81,3 +81,85 @@ font-family: "Noto Sans";
 font-size: 12px;
 }
 
+.sb-select-with-search .searchbox-input,
```

Comment:
Are there any rules applied to the existing elements - or are all contingent on `isSearchable`?